buildtools/container/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.16.0
+FROM node:10.16.0
```

Comment:
I think we probably want to use Node 14.17.3 (LTS). that is the one recommended for most users.
https://nodejs.org/en/about/releases/

Reply:
ok